cloneContribFiles.sh

```diff
@@ -0,0 +1,4 @@
+rm -rf src/main/groovy/edu/oregonstate/mist/contrib/
+git clone --depth=1 -q git@github.sig.oregonstate.edu:ecs-data/id-images-api-contrib.git src/main/groovy/edu/oregonstate/mist/contrib
```

Comment:
We want this script to be generic so that it can be included in the skeleton. Let's define the git url and repo in the gradle.properties. We should also have a flag that enables / disables cloning contrib files.